fullstack_template/server/server.py

```diff
@@ -75,33 +75,52 @@ def print_all_users():
             db.session.rollback()
     return "success"
 
-@app.route('/orgs',methods=['POST'])
+@app.route('/orgs/create',methods=['POST'])
 @jwt_required()
 def create_organization():
     request_data = request.get_json()
 
     org_type = OrganizationType.query.filter_by(name=request_data["organizationType"]).first()
-    data = Organization(name=request_data["organizationName"], 
-                        description=request_data["organizationDescription"])
-    data.org_type = org_type
-    data.admins = [current_identity]
+    org = Organization(name=request_data["organizationName"], 
+                        description=request_data["organizationDescription"],
+                        image=request_data["organizationImage"])
+    org.org_type = org_type
+    org.admins = [current_identity]
     
     try:     
-        db.session.add(data)
+        db.session.add(org)
         db.session.commit()
     except:
         db.session.rollback()
 
     return jsonify(message="successful organization creation")
 
+#adds the current user as a member of the specified organization
+@app.route('/orgs/join',methods=['POST'])
+@jwt_required()
+def join_organization():
+    request_data = request.get_json()
+
+    #org = Organization.query.get(request_data["organizationId"])
+    org = Organization.query.get(1)
```

Comment:
We will switch swap this line with the previous line when the frontend for joining an organization is implemented.